datasets/_global/gem_energy_ownership/crawler.py

```diff
@@ -17,15 +17,15 @@
     "gem_parents",
     "gem_parents_ids",
 ]
-ALIAS_SPLITS = ["[former],", "[former]", "[former name]", "(former)"]
+ALIAS_SPLITS = ["[former],", "[former]", "[former name]", "(former)", ", "]
```

Comment:
splitting company names on comma is often a pain :'(

Haven't found more than this example yet, but I think we're going to have to accept splits using lookups until zavod ui is up. 

You can try asking gpt to do the splitting and spit out yaml so the accepted values can be listed in lookups more easily. 


![image](https://github.com/user-attachments/assets/19e1c2f5-b22b-4e80-a40a-11169f1cca23)


You could also maybe do the lookup like

```yaml
- match: "Arkhangel'skiy Tsellyulozno-Bumazhnyi Kombinat AO, Arkhangel'skiy TsBK"
  split: true
- match: "Armada, AO"
  values: ["Armada, AO"]
```

so that the values that can be autosplit don't have to be repeated.